sourcemap/fleet_store_test.go

```diff
@@ -77,4 +77,106 @@ func TestFleetFetch(t *testing.T) {
 	assert.True(t, hasAuth)
 }
 
+func TestMultipleFleetHostsFetch(t *testing.T) {
```

Comment:
Seeing as we're not testing which server is used, maybe we should fold this into TestFleetFetch above - just update that test to have two servers?